sympy/functions/elementary/miscellaneous.py

```diff
@@ -337,7 +337,6 @@ def __new__(cls, *args, **assumptions):
 
         # second filter
         # variant I: remove ones which can be removed
-        # args = cls._collapse_arguments(set(_args), **assumptions)
```

Comment:
Though this function doesn't exist, perhaps this should still be left as a reminder for when it did.